src/winrt/example/TestFiles/demo2.manifest

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0">
+  <file name="test_component.dll">
+    <comClass
+        clsid="test_component.Class"
+        threadingModel="Both">
+
+      <progid>"test_component.Class"</progid>
```

Comment:
Quotes shouldn't be required within the `<progid/>` element.

Reply:
Sorry, wasn't actually using the progid in my test.  Holdover from copy/paste.  Will remove from test manifest.